### PR TITLE
Change VimL to Vimscript

### PR DESCRIPTION
**Pull request by UltiRequiem**

### CODE REVIEW

**Comment by bfredl** [comment]
I don't think editing past newsletters make sense.

File: _posts/2014-11-07-november-newsletter.md [hunk 1]
```diff
@@ -97,15 +97,15 @@ The merging of this latest pull request chips away at the long standing
...
-@ZyX-I is still hard at work on advancing [his VimL to Lua
-translator][viml-translator]. More than **80%** of Vim commands have been
+@ZyX-I is still hard at work on advancing [his Vimscript to Lua
+translator][vimscript-translator]. More than **80%** of Vim commands have been
 implemented in the parser.
 
 In the process of writing the translator, he has come across various bugs
-([1][viml-1], [2][viml-2], [3][viml-3], [4][viml-4], [5][viml-5]). He continues
-to report them to the [vim\_dev][viml-dev] mailing list as well as contributing
+([1][vimscript-1], [2][vimscript-2], [3][vVimscriptimscript-3], [4][vimscript-4], [5][vimscript-5]). He continues
```
Comment (mhinz) [inline _posts/2014-11-07-november-newsletter.md:107]: Typo.

**Comment by mhinz** [comment]
Personally I always preferred the shorter "VimL", but I'm really passionless about this change.

**Comment by UltiRequiem** [comment]
Should I close this pull request?

**Comment by justinmk** [comment]
> Should I close this pull request?

as bfredl said, the old newsletters should be left alone. but the other changes make sense.

**Comment by UltiRequiem** [comment]
Please review #236.

**Pull request closed without merging**